ground/src/main/java/com/google/android/ground/persistence/local/room/converter/ConverterExt.kt

```diff
@@ -112,7 +112,14 @@ fun JobEntityAndRelations.toModelObject(): Job {
     jobEntity.style?.toModelObject(),
     jobEntity.name,
     taskMap.toPersistentMap(),
-    jobEntity.suggestLoiTaskType?.let { Task.Type.valueOf(it) }
+    jobEntity.suggestLoiTaskType?.let {
+      try {
+        Task.Type.valueOf(it)
+      } catch (e: Exception) {
+        Timber.e("unknown task type: $it")
+        Task.Type.UNKNOWN
```

Comment:
I think it would be more correct here to skip the task rather than create a task with an "UNKNOWN" task type, since those tasks will be inactionable downstream.

Reply:
Done